.gitpod.yml

```
@@ -0,0 +1,7 @@
+tasks:
+  - name: Initialize Poetry Environment
```

This will become the name of the terminal itself, is it suitable for that though?

---

I think so

---

I'd suggest something more like just a bare:
```suggestion
  - name: Python Environment
```

---

I'll do this, but to clarify this only sets this as the terminal name while the commands are running